### DISCUSSION TITLE
Chart labels hard to see in dark mode admin

### DISCUSSION THREAD

**Discussion by ziming** [filamentphp/filament#2859] [Help]
### Package

filament/filament

### Package Version

2.13.8

### Laravel Version

8

### Livewire Version

2

### PHP Version

8.1

### Bug description

The grey labels r very hard to see especially pie chart labels where the text is even smaller and gray. It will be better if the labels are whiter

![image](https://user-images.githubusercontent.com/679513/174723172-8a1a392f-3733-4876-9f02-22402d3017c3.png)

**Comment by Z3d0X** [comment]
You can provide custom colors.
 Filament uses chartjs for charts. I suggest you take a look at [Chartjs Documentation](https://www.chartjs.org/docs/latest/charts/bar.html)

Here is an example to customize color:
```php
protected function getData(): array
  {
      return [
          //..
          'datasets' => [
              [
                  //...
                  'backgroundColor' => [
                      'rgba(255, 99, 132, 0.5)',
                      'rgba(255, 159, 64, 0.5)',
                      'rgba(255, 205, 86, 0.5)',
                      'rgba(75, 192, 192, 0.5)',
                      'rgba(54, 162, 235, 0.5)',
                      'rgba(153, 102, 255, 0.5)',
                      'rgba(201, 203, 207, 0.5)'
                  ],
                  'borderColor' => [
                      'rgb(255, 99, 132)',
                      'rgb(255, 159, 64)',
                      'rgb(255, 205, 86)',
                      'rgb(75, 192, 192)',
                      'rgb(54, 162, 235)',
                      'rgb(153, 102, 255)',
                      'rgb(201, 203, 207)'
                  ],
                  'borderWidth' => '1',
              ]
          ]
      ];
  }
```

**Comment by ziming** [comment]
Is your example about the labels? I'm not talking about the pie or bar colours

**Comment by Z3d0X** [comment]
> Is your example about the labels? I'm not talking about the pie or bar colours

Ahh sorry for misunderstanding.

I'm not sure how to do that exactly, but presume its within the chart options? You can provide custom options via this method.
```php
protected function getOptions(): ?array
  {
      return [
          'indexAxis' =>  'x', //just an example
      ];
  }
```
